tests/run-make/allow-warnings-cmdline-stability/rmake.rs

```diff
@@ -0,0 +1,18 @@
+// Test that -A warnings makes the 'empty trait list for derive' warning go away
+
+use run_make_support::rustc;
+
+fn grep(out: String, look_for: &str) {
+    if out.contains(look_for) {
+        panic!("{:?} was found in {:?} whereas it shouldn't have!", look_for, out);
+    }
+}
```

Comment:
Suggestion: `grep` reads a bit awkward, maybe something like

```rs
fn assert_not_contains(haystack: &str, needle: &str) {
    if haystack.contains(needle) {
        eprintln!("=== HAYSTACK ===");
        eprintln!("{}", haystack);
        eprintln!("=== NEEDLE ===")
        eprintln!("{}", needle);
        panic!("needle was unexpectedly found in haystack");
    }
}
```

This could even be a helper fn in the support library

Reply:
Let's rename and move it to the support library then!